modules/userId/index.js

```diff
@@ -1248,12 +1268,17 @@ export function init(config, {mkDelay = delay} = {}) {
     if (userSync) {
       ppidSource = userSync.ppid;
       if (userSync.userIds) {
+        const {autoRefresh = false, retainConfig = true} = userSync;
```

Comment:
I know the defaults are my idea, but could you check if they match what's currently in master? retainConfig = true means that it should be the current behavior (or a breaking change), but that doesn't seem to be the case judging from your changes here.

Reply:
actually , I missed that this is for 10, so the breaking change is OK.